### PR TITLE
Dragengine 1.11-haiku1

### PR DESCRIPTION
**Pull request by LordOfDragons**
Fix added. See conversation in #6776

### CODE REVIEW

**Comment by Begasus** [comment]
Conflicting files, I guess you didn't delete/recreate the branch?

**Comment by LordOfDragons** [comment]
I don't know. First time I have this issue. But using REVISION=2 version solves the conflict and all should be fine. If it doesn't work I can try recreate the branch. Something did not work as it usually did :(

**Comment by Begasus** [comment]
1. create a branch
2. create a PR
3. after merge delete remote and local branch
4. make sure master is on par with upstream (local/remote)
5. create new PR ...
That's usually my flow (but granted I miss sometimes too) :)

File: games-engines/dragengine/dragengine-1.11.recipe [hunk 1]
```diff
@@ -14,11 +14,11 @@ HOMEPAGE="https://dragondreams.ch
 COPYRIGHT="2020-2021 DragonDreams"
```
Comment (diversys) [inline games-engines/dragengine/dragengine-1.11.recipe:14]: Bump

**Comment by kenmays** [comment]
[games-engines/dragengine/dragengine-1.11.recipe](https://github.com/haikuports/haikuports/pull/6778/files/8d3fc9bf17e86ee04962769cf0610864a923cc0a#diff-ee9939d89159fa6199eb69b4a76b53ec7bb4618a3fd61738c09c71987bfa25ee)

Fixes:
1. Update COPYRIGHT.
2. Under LICENSE, tabulate additional licenses for proper formatting (as shown).
3. For ARCHITECTURES, use 'all' as a default for targeting the modern Haiku x86/x64 platforms for build.

`COPYRIGHT="2020-2022 DragonDreams"`

```
LICENSE="GNU GPL v3
	GNU LGPL v3"
```

`ARCHITECTURES="all !x86_gcc2"`

**Comment by kenmays** [comment]
x86 build failure:
```
Compiling src/launcher/console/src/consolelauncher.cpp
src/launcher/console/src/consolelauncher.cpp:54:10 fatal error: engine/declEngine.h: No such file or directory

#54 | #include "engine/declEngine.h"
```

NOTE: https://github.com/LordOfDragons/dragengine/blob/master/src/launcher/console/src/consolelauncher.cpp

**Comment by LordOfDragons** [comment]
I'll recreate the branch to apply the fixes.

**Pull request closed without merging**